mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -191,6 +200,15 @@
       >
         Set to <Null />
       </ButtonMenuItem>
+      <MenuDivider />
+      <!-- Column Attributes -->
+      <ColumnHeaderContextMenu {processedColumn} />
+      <!-- Column Attributes end -->
+      <MenuDivider />
+      <!-- Row -->
+      <RowContextOptions recordId={Number(rId)} {recordsData} {row} />
```

Comment:
The `recordId` is not always a number. This is an existing bug in the `RowContextOptions` component. Change the property type from `recordId: number` to `recordPk: string`.

This line needs to change as:
```suggestion
      <RowContextOptions recordPk={String(recordPk)} {recordsData} {row} />
```

Similarly, also make changes in the `Row` component where this component (`RowContextOptions`) is used.

If you want to raise an issue for this, please go ahead, as this is not strictly related to the issue this PR addresses.

Reply:
recordid is not defined as string because it's type is number in RowContextOptions.svelte file.
Do you want me to change it there? because it may raise more errors elsewhere where we are passing it as in Number format.

Reply:
@TafreedAhmad 

Yes, as mentioned in the review comment, please make the change in `RowContextOptions`.

> because it may raise more errors elsewhere where we are passing it as in Number format.

It is only used in one other place, which is also mentioned in the above comment i.e. the `Row` component.